tests/test_filesize.py

```diff
@@ -0,0 +1,15 @@
+import os
+from os.path import getsize
+
+
+dir = ('safebrowsing')
+filenames = []
+for file in os.listdir(dir):
+        if os.path.isfile(os.path.join(dir, file)) and 'mozstd-trackwhite' in file:
+            size = os.path.getsize(os.path.join(dir, file))
+            if size <= 200000:
```

Comment:
I'm not sure how I'd do that as the maximum size will be different depending on the list/file type.